spec/spec_helper.rb

```diff
@@ -23,6 +25,11 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  config.before(:each) do
```

Comment:
Yes it is needed for the details_spec feature as the loading of the documents checklist step triggers a documents listing to show in the view any previously uploaded docs.

At the moment it's true there are few places and we could stub the uploader directly in each of those places, but I think there will be more places soon once we start writing more feature tests, and other than in some specific tests, we probably want to just stub it as a global default.

But I'm open to change it.